### PR TITLE
Added Fission under Integrations (under Ecosystem)

### PR DESCRIPTION
**Pull request by techmaharaj**
Hi Team,
I'd like to update Fission under Integrations. We are a serverless framework for Kubernetes and have recently added integration of Istio. You can check out more details at the below link:

https://fission.io/docs/usage/ingress/ingress-istio/

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by craigbox** [comment]
/ok-to-test

LGTM, thanks, and welcome to the Istio family!

File: data/companies.yml [hunk 1]
```diff
@@ -169,6 +169,12 @@ integrations:
     description: "Flagger is a Progressive Delivery Operator for Kubernetes."
     details:
       - "Flagger can run automated application analysis, testing, promotion and rollback for Canary (progressive traffic shifting), A/B Testing (HTTP headers and cookies traffic routing) and Blue/Green (traffic switching and mirroring) using Istio."
+  - name: "Fission"
+    logo: "/logos/fissionio.svg"
+    url: "https://fission.io/docs/usage/ingress/ingress-istio/"
+    description: "Fission is a framework for serverless functions on Kubernetes. "
+    details:
+      - "Fission allows users to deploy an Istio Gateway to route traffic to fission functions. Users can also authenticate requests to fission functions using Istio's authentication capabilities."
```
Comment (craigbox) [inline data/companies.yml:177]: flagging that you wrote "fission" and you might prefer to have written "Fission".  Can't tell from your docs if you have a strong branding preference either way.

Comment (techmaharaj) [reply data/companies.yml]: Thanks Craig for pointing that out, have updated it.

**Review by craigbox — APPROVED** [review]
/cherrypick release-1.13

**Pull request merged**